### PR TITLE
exclude lxml 4.7 because the wheels are missing the lxml includes

### LINKED ISSUE #198: Fails to build "lxml-version.h: No such file or directory"
Suddenly failed to build recently. Might be due to the new release of lxml that came out

```#11 22.22   In file included from /tmp/pip-install-w06wmd73/xmlsec_b455cf60a688463dbc08380657687275/src/ds.c:15:
#11 22.22   /tmp/pip-install-w06wmd73/xmlsec_b455cf60a688463dbc08380657687275/src/lxml.h:18:10: fatal error: lxml-version.h: No such file or directory
#11 22.22      18 | #include <lxml-version.h>
#11 22.22         |          ^~~~~~~~~~~~~~~~
#11 22.22   compilation terminated.
#11 22.22   error: command 'gcc' failed with exit status 1
```

### PR DESCRIPTION
**Pull request by graingert**
fixes #198

### CODE REVIEW

**Comment by hoefling** [comment]
@graingert to fix the pipeline, can you also adjust the `lxml` range in `requirements.txt`? It's used in the CI to install dependencies. BTW an awesome idea about adding `pkgconfig` to build deps - we can now drop it from `requirements.txt` as well, since it was never a runtime dependency anyway.

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 pkgconfig
```
Comment (graingert) [inline requirements.txt:1]:
```suggestion
```

**Comment by graingert** [comment]
I set the dep to `!=4.7.0` because the missing includes appears to be a release process bug and not an intentional remove and so a `4.7.1` release could fix this

**Comment by graingert** [comment]
`setup.py bdist_wheel` is deprecated and doesn't support pyproject.toml build-system requires

https://web.archive.org/web/20211121161231/https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html#summary

**Comment by graingert** [comment]
> @graingert to fix the pipeline, can you also adjust the `lxml` range in `requirements.txt`? It's used in the CI to install dependencies. BTW an awesome idea about adding `pkgconfig` to build deps - we can now drop it from `requirements.txt` as well, since it was never a runtime dependency anyway.

done

**Comment by hoefling** [comment]
@graingert awesome, thank you for the quick fix! I will try to release it today when I'm back from work.

**Pull request merged**